### PR TITLE
minor bug fixes

### PR DESCRIPTION
**Pull request by Nayananga**

### CODE REVIEW

File: docs/components/modules/ROOT/pages/aws-s3-component.adoc [hunk 1]
```diff
@@ -87,7 +87,7 @@ with the following path and query parameters:
 [width="100%",cols="2,5,^1,2",options="header"]
 |===
 | Name | Description | Default | Type
-| *amazonS3Client* (common) | Reference to a com.amazonaws.services.s3.AmazonS3 in the xref:registry.adocRegistry. |  | AmazonS3
+| *amazonS3Client* (common) | Reference to a com.amazonaws.services.s3.AmazonS3 in the xref:registry.adoc [Registry]. |  | AmazonS3
```
Comment (oscerd) [inline docs/components/modules/ROOT/pages/aws-s3-component.adoc]: This is part of the javadoc.. you need to modify the javadoc and rebuild

**Comment by oscerd** [comment]
The autogenerated part of the doc must be modified in the javadoc

File: docs/components/modules/ROOT/pages/aws-s3-component.adoc (hunk 1 above)
Comment (Nayananga) [reply docs/components/modules/ROOT/pages/aws-s3-component.adoc]: Thank you Mr.Andrea, Did you mean to use mvn package command?

**Comment by oscerd** [comment]
Mvn clean install on the component.. you need to change the S3Configuration class first, by modifying the javadoc and after that, run mvnclean install

File: docs/components/modules/ROOT/pages/aws-s3-component.adoc (hunk 1 above)
Comment (Nayananga) [reply docs/components/modules/ROOT/pages/aws-s3-component.adoc]: when I use mvn package command every change I did revert back to where I was before.How can I fix that?

Comment (oscerd) [reply docs/components/modules/ROOT/pages/aws-s3-component.adoc]: You need to edit the javadoc related to that parameter in the S3Configuration Java class

Comment (Nayananga) [reply docs/components/modules/ROOT/pages/aws-s3-component.adoc]: Okay, I found the reason from camel-website readMe. But still not clear about javadoc thing,

Comment (oscerd) [reply docs/components/modules/ROOT/pages/aws-s3-component.adoc]: If you modify the s3configuration class, you need to rebuild the module components/camel-aws-s3 with mvn clean install. After that you can go back to docs folder and run mvn clean package to have the updated adoc under ROOT

Comment (Nayananga) [reply docs/components/modules/ROOT/pages/aws-s3-component.adoc]: Okay, got it, so do i have to do this for each of every component I`ve changed?

Comment (oscerd) [reply docs/components/modules/ROOT/pages/aws-s3-component.adoc]: Yes.

**Comment by Nayananga** [comment]
#!/bin/bash
function copyContent() {
    for filename1 in /home/nayananga/Desktop/camel/docs/components/modules/ROOT/pages/*.adoc; do
        extractName1=${filename1##*/}
        find '/home/nayananga/Desktop/camel/components/' -iname "*.adoc" -printf "%h\n" | sort -u | while read i; do                                              
            cd "$i" 
            for filename2 in $PWD/*.adoc; do
                extractName2=${filename2##*/}
                if [ "$extractName1" == "$extractName2" ];then                 
                    echo "match found coping $filename1 to $filename2"
                    cp $filename1 $filename2
                    runmvncleaninstall
                fi
            done
            # && pwd && $COMMAND
        done    	
    done
}

function runmvncleaninstall () {
        cd ../../../
        mvn clean install
}
copyContent
exit 0

**Comment by Nayananga** [comment]
I wrote a little script to solve  the issue, I think It might fix the problem, It still running on my pc

**Comment by Nayananga** [comment]
Hello Mr.Andrea and Mr.Zoran, can you check now? but still, I fixed the autogenerated files in camel/components when I run mvn clean package inside camel/docs, the user-manual files revert back. I couldn`t find the reason for that. Any suggestions?

**Comment by davsclaus** [comment]
Thanks for this PR. We can later work on the links inside those auto-generated parts (eg see the // START // END comments in the adoc file that mentions that the section is auto-generated and that you should not edit it directly there.

**Comment by davsclaus** [comment]
Okay merged, lets do new PRs for new work.

**Pull request merged**